localization/zh-TW/strings.json

```diff
@@ -110,7 +123,8 @@
     "reviews": "評論",
     "read_reviews": "閱讀評論",
     "read_more_reviews": "閱讀更多評論於",
-    "gameplay": "實況",
+    "gameplay": "",
```

Comment:
Why is that string empty?

Reply:
I left it blank because I think searching with the name + PC is sufficient for gameplay videos. I found searching with an English app name + translated keyword causes a lot of smaller games to display no results. Larger games won't necessarily get the best results either since videos catered to a local audience will use translated names in the title. Showing some English videos is better than nothing IMO.

Reply:
Such logic shouldn't be done in a localization file though.
And also, if the user wants to see English videos, why isn't he browsing Steam in English?

Reply:
Just like steam does, English could be the user's secondary language. English videos have a larger quantity than localized videos.

Reply:
That's true, but I expect YouTube to be smart enough to curate the most appropriate videos

Reply:
Yeah youtube serves content based on location, so I think searching with English only is best: you get both English and local content. But okay I'll put back a term that's fits better. The problem isn't with localization, it's with limiting the search to videos containg an English name + a specific keyword in the title.